presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClientConfig.java

```diff
@@ -379,4 +394,16 @@ public CassandraClientConfig setSpeculativeExecutionDelay(Duration speculativeEx
         this.speculativeExecutionDelay = speculativeExecutionDelay;
         return this;
     }
+
+    public ProtocolVersion getProtocolVersion()
+    {
+        return protocolVersion;
+    }
+
+    @Config("cassandra.protocol-version")
+    public CassandraClientConfig setProtocolVersion(String version)
+    {
+        protocolVersion = ProtocolVersion.valueOf(version);
```

Comment:
`this.protocolVersion = ..` (even though this is not technically necessary)

Reply:
This is something we should explicitly declare in our coding style. I'm also big fan of adding `this.` prefix because it is less error prone. But several times i got review comments to remove the `this.` prefix. So, we should came into some conclusion here. @electrum @nezihyigitbasi

Reply:
I didn't mean to start a broader discussion. I just think a setter is usually in form of `this.field = ...`. In general -- I think it might actually vary (though some guidance would be welcomed)